pages/getting-started.mdx

```diff
@@ -96,11 +96,10 @@ You can connect to the Memgraph instance using the command-line interface
 mgconsole by running the following command in a new terminal:
 
 ```terminal
-# use `docker ps` to get the Memgraph container id for memgraph-mage
-docker exec -ti <container-id> mgconsole
+docker run -it memgraph/mgconsole:latest --host HOST_IP --port PORT
 ```
 
-When the mgconsole starts, you can query the database. 
+If you're running Memgraph on your local machine, `--host` and `--port` are not necessary.
```

Comment:
Let's please invert the sense here. Something like:
```
The arguments --host and --port are only required when connecting to a remote Memgraph server. Otherwise, they can be omitted.
```